tests/gateway_with_yggdrasil_ip/gatway_with_yggip_test.go

```diff
@@ -1,10 +1,7 @@
-//go:build integration
-// +build integration
-
 package test
 
```

Comment:
the whole scenario is problematic, needs to be rethought, should be as easy as running nginx or any server on the machine, create a name contract, curl that domain make sure the word nginx appears in the output